about.md

```diff
@@ -4,15 +4,17 @@ title: About
 ---
 # About us
 
-Welcome to the **Computer Science Society**! We are the departmental society
-in the School of Computer Science at the University of Birmingham.
+Welcome to the **Computer Science Society**! We are the departmental society in the School of Computer Science at the University of Birmingham.
 
-We are one of over 300 Guild of Students societies and one of 35+ EPS College
-societies. Being a student is not just about working hard, you have to enjoy
+We are one of over 300 Guild of Students societies and one of 35+ EPS College societies. Being a student is not just about working hard, you have to enjoy
 it too! That is why CSS is working making sure to create events and provide a
 medium through which computer scientists can meet, make friends, and partake
 in a whole lot of exciting activities.
 
+## Constitution
+
+As a society with the Guild of Students, we have a constitution that dictates our structure, how we meet, come to decisions, etc. You can find it [here][constitution]. All our committee signs the constitution, and we encourage all our members to read it at least once.
```

Comment:
Where's the actual constitution? Should it be a PDF or as another page?

Reply:
It's linked at the bottom of the markdown file, as a google drive link.

Reply:
That'll work for now. But should probably create a dedicated page for it (e.g. /about -> /about/constitution) before the AGM.

Reply:
Nice, good idea, I'll add a tracking issue.